### PR TITLE
feat(troubleshooter): added basic troubleshooter

### PR DESCRIPTION
**Pull request by ravindra-dyte**
| Linear Issue |
| :----------: |
| fixes WEB-4078 |

### Description

Added basic troubleshooter.

### Changes needed

1. Will be adding Systems tab
2. Will be adding CPU & bandwidth limitations indicator.
3. Latency & Speaker delay (Phase 2)
4. Need to add i18n.

### Screenshots

image
image
image
image
image
image

### CODE REVIEW

File: packages/core/src/utils/debugger-utils.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { MediaKind } from '@dytesdk/web-core';
+
+export type StatsHealth = 'Good' | 'Normal' | 'Worst';
```
Comment (palashgo) [inline packages/core/src/utils/debugger-utils.ts:3]: Can you use `Good` > `Average` > `Poor` for the levels

File: packages/core/src/utils/debugger-utils.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+export function getPacketLossVerdict({
+  packetLossPercentage,
+}: {
+  packetLossPercentage: number;
+}): StatsHealth {
+  let verdict: StatsHealth = 'Good';
+
+  if (packetLossPercentage > 5) {
+    verdict = 'Worst';
+  } else if (packetLossPercentage > 0) {
```
Comment (palashgo) [inline packages/core/src/utils/debugger-utils.ts:45]: 0 - 1 - Good 
1 - 4 - Average
4 + - Poor

File: packages/core/src/utils/debugger-utils.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  return verdict;
+}
+
+/** Gives verdict based on the jitter */
+export function getJitterVerdict({ jitterInMS }: { jitterInMS: number }): StatsHealth {
+  let verdict: StatsHealth = 'Good';
+
+  if (jitterInMS > 300) {
+    verdict = 'Worst';
```
Comment (palashgo) [inline packages/core/src/utils/debugger-utils.ts:57]: 0 - 50 - good
50 - 100 - average
100+ - poor

Comment (ravindra-dyte) [reply packages/core/src/utils/debugger-utils.ts]: Done.

**Review by palashgo — APPROVED** [review]

**Pull request merged**